go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -1414,6 +1414,21 @@ var DiffSystemTableScriptTests = []enginetest.ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "table with commit column should maintain its data in diff",
+		SetUpScript: []string{
+			"CREATE TABLE t (pk int PRIMARY KEY, commit text);",
+			"set @Commit1 = dolt_commit('-am', 'creating table t');",
```

Comment:
why do we `set @Commit1` and `set Commit2`?

Reply:
we don't need them. replaced with `CALL dolt_commit`